src/Extension/TemplateExtension.php

```diff
@@ -59,6 +64,13 @@ public function getTokenParsers(): array
      */
     public function getUrlsafeIdentifier($model): ?string
     {
+        @trigger_error(sprintf(
+            'Method "%s()" is deprecated since sonata-project/twig-extension 1.7 and will be removed in version 2.0'
+            .' in favor of the "sonata_urlsafeid" method of the SonataAdminBundle.'
```

Comment:
```suggestion
            'Method "%s()" is deprecated since sonata-project/twig-extension 1.7 in favor of the "sonata_urlsafeid"'
            .' Twig filter defined by SonataAdminBundle and will be removed in version 2.0.'
```